index.js

```diff
@@ -143,3 +153,16 @@ MultiStream.obj = streams => (
 )
 
 module.exports = MultiStream
+
+// Normalize stream destroy w/ callback.
+function destroy (stream, err, cb) {
```

Comment:
if the stream is already destroyed you need to return early i think

Reply:
not for native `destroy`, however, overriden `destroy` might not behave as expected so it's probably a good idea

Reply:
fixed